roles/lnls-ans-role-sirius-hla/tasks/setup-apps.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Clone and Install Applications
+  include_tasks: clone-install-app.yml
+  with_items:
+    - "{{ sirius_apps }}"
```

Comment:
Wrong variable name, should be `hla_apps`

Reply:
done